bot.py

```diff
@@ -47,6 +48,46 @@ def does_channel_exist(channel_name):
         return False
 
 
+def prune_channels(user_token):
+    response = client.channels_list()
+    channels = response["channels"]
+    for channel in channels:
+        if channel["name"].startswith("111-change-"):
+            channel_id = channel["id"]
+            requests.get(f"https://slack.com/api/channels.delete?token={user_token}&channel={channel_id}")
+    return make_response("", 200)
+
+def prune_ask_token(trigger_id):
+    view_open = client.views_open(
+        trigger_id=trigger_id,
+        view={
+            "type": "modal",
+            "callback_id": "prune_ask_token_modal",
+            "title": {
+                "type": "plain_text",
+                "text": "Prune changes"
+            },
+            "submit": {"type": "plain_text", "text": "Prune"},
+            "close": {"type": "plain_text", "text": "Cancel"},
+            "blocks": [
+                {
+                    "block_id": "user_token",
+                    "type": "input",
+                    "label": {
+                        "type": "plain_text",
+                        "text": "What is your legacy user token?"
+                    },
+                    "element": {
+                        "type": "plain_text_input",
+                        "action_id": "txt_user_token"
+                    },
+                }
+            ],
+        },
+    )
+    
+    return make_response("", 200)
```

Comment:
Actually same here - you don't need to return this make_response object for a normal function

Python is quite happy with no return statement, if you have nothing to return (equivalent of void I suppose)

Or you can just put `return` as the return line - it will return None